app/models/account_decorator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module AccountDecorator
+  MOBIUS_TENANTS = %w[
```

Comment:
@kirkkwang this whole decorator can be deleted now, right?

Reply:
We may need some way to identify the mobius tenants. It's not used but i think that's why it was left.

Reply:
I thought we decided we didn't need to do so in this way? Even if we do end up needing to, the code is in git, so we can easily restore it. In my opinion, removing it now means there's no risk of forgetting to do so later. A `TODO` comment could work if we really want to keep it for now.

Reply:
I think ultimately we'll remove it... I think we were going to just add it into a readme or something or change it to a TODO but I don't see that has happened yet.